src/cc/api/BPFTable.cc

```diff
@@ -106,6 +106,12 @@ BPFStackTable::~BPFStackTable() {
     bcc_free_symcache(it.second, it.first);
 }
 
+void BPFStackTable::clear_table_non_atomic() {
+  for (int i = 0; i < capacity(); i++) {
+    remove(&i);
+  }
```

Comment:
stack table essentially is a hash table with a key of 32-bit int. There are two ways to clear the table. One is the above, and the other is to do similar to hash table, using bpf_get_first_key and bpf_get_next_key API. Do you have particular reason to choose this one?

Reply:
@yonghong-song I thought so too, but: https://github.com/torvalds/linux/blob/master/kernel/bpf/stackmap.c#L231

Reply:
Okay, then the change looks good. I will check the kernel side to see whether we can implement get_next_key API for stacktrace map.